cli/flox/src/commands/init/python.rs

```diff
@@ -1049,12 +1053,7 @@ mod tests {
     /// None should be returned for an empty pyproject.toml
     #[tokio::test]
     async fn poetry_pyproject_empty_with_catalog() {
-        let (mut flox, _temp_dir_handle) = flox_instance();
-
-        if let Client::Mock(ref mut client) = flox.catalog_client {
-            // Response for no package groups
-            client.push_resolve_response(vec![]);
-        }
+        let (flox, _temp_dir_handle) = flox_instance();
```

Comment:
question nonblocking: was pushing the `vec![]` unnecessary?

Reply:
We already panic if we attempt resolution without pushing a resolve response. If you push a `vec![]` you end up with `packages: []` in the lockfile. So I think you just strengthened the test which is good.

I'm not sure yet if we'll want to completely drop `push_resolve_response`, but I don't think that will impact this test.